codes/quantum/qubits/surface/triangle.yml

```diff
@@ -0,0 +1,56 @@
+    - 'The basis-state conversion (BC) initialization and measurement method initializes and measures a Pauli eigenstate. To do this, triangle ancilla qubits are required outside of the triangle patches that hold the data. That is, the ancilla patches must be empty of data and be adjacent to the side that contains the logical Pauli that is wanted to be measured or initialized. The method uses fewer qubits than the CC method to convert a basis state from the surface code to the triangle code. It also uses fewer qubits than the CC method to measure the basis-state transversally. This method preps a basis state in a \(d \times d\) surface code as the CC method except now instead of treating the surface code as \(1/3\) of a triangle patch, it is \(2/3\). To convert to the surface code, stop measuring the bottom loop operators of the patch and measure the stabilizers of a \((d+1)/2 \times (d+1)/2 \times d\) triangle code \(O(d)\) times. Then, symmetrize the triangle code by measuring the stabilizers and not measuring the stabilizers in the topmost rows. This procedure requires \(O(1)\) time to perform \(H,S\) gates and \(O(d)\) time to perform \(CNOT\). Sometimes reorientation of the sides is required and that takes \(O(d)\) to perform \cite{doi:10.22331/q-2017-04-25-2}.'
+    - 'The CAT states (CS) initialization and measurement method uses a row of \(d\) ancilla qubits along some edge of a triangle code with distance \(d\) to create and verify a CAT state that is to measure the logical operator along the same edge. Creating this CAT state takes \(O(d)\) time steps. To relaibly measure the logical state, the CAT state must be measured \(O(d)\) times resuling in \(O(d^2)\) time for logical measurement. Initialization is a similar procedure that requires \(O(d^2)\) time for logical operator measurements that occur \(O(d)\) times as well as \(O(d^2)\) time to project the code onto a logical state \cite{doi:10.22331/q-2017-04-25-2}.'
+
+  transversal_gates: 'Triangle codes have transversal single-qubit Clifford gates of order \(3\) like Clifford gate \(\bar{SH}\) that performs \(H\) on all qubit positions \(x > 0\) or at \(x = 0\) and \(z > 0\) and then perform \(SH\) on the central qubit. Note that qubits are places on integer lattics in the \(xy-, yz-,\) and \(xz-\) planes \cite{doi:10.22331/q-2017-04-25-2}.'
+
+  #general_gates:
+
+  decoders:
+    - 'For decoding a single layer of the triangle code, a decoding graph is laid on the code. Nodes of the graph are located at each stabilizer (center of the triangle graph) and has red or blue edges where red associates with \(X\) errors and blue with \(Z\) errors. To take into account any errors from measuring the error syndrome, a three-dimensional stack of the decoding graphs is laid ontop of the code with vertical edges connecting to qubits between layers \cite{doi:10.22331/q-2017-04-25-2}.'
```

Comment:
Any comments on decoding complexity would be great to add here. I'm assuming it's comparable to surface code?